x-pack/legacy/plugins/infra/public/pages/logs/analysis/page_results_content.tsx

```diff
@@ -180,7 +188,33 @@ export const AnalysisResultsContent = ({ sourceId }: { sourceId: string }) => {
             <EuiPageBody>
               <EuiPageContent>
                 <EuiPageContentBody>
-                  <LogRateResults isLoading={isLoading} results={logEntryRate} />
+                  {isFirstUse ? (
```

Comment:
Could we solve the "dismiss the callout" problem by using something like
```typescript
isFirstUse && !hasResults
```
as the condition where
```typescript
const hasResults = memo(() => logEntryRate.histogramBuckets.length > 0, [logEntryRate.histogramBuckets]);
```
?

That would cause the callout to disappear once there are log rate data.

Reply:
Ah, yes, that would work. Once we have 2 - 3 sections I might move that in to the results hook itself, but it's okay where it is for now.